sale_stock_picking_update_responsible/models/stock_picking.py

```diff
@@ -0,0 +1,23 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, fields, api
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    is_user_id_editable = fields.Boolean(
+        compute="_compute_is_user_id_editable",
+    )
+
+    @api.depends("state")
```

Comment:
There's another field that is used within compute method and should be added to `depends` clause, please check.

Not sure if compute is recomputed with a user change, we should make a test, we'll talk about it